_rules/descriptive-link-5effbb.md

```diff
@@ -0,0 +1,292 @@
+---
+id: 5effbb
+name: Link in context is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the accessible name of a link together with its context describe its purpose.
+accessibility_requirements:
+  wcag20:2.4.4: # Link Purpose (In Context)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
```

Comment:
I don´t think this rule satisfies.

This rule together with https://act-rules.github.io/rules/c487ae does satisfy. But currently a link can have an empty accessible name and be marked inapplicable while it would fail SC 2.4.4.

Either this rule doesn't satisfy completely or this rule needs an assumption that all links have a non-empty accessible link. Which I don't think is a good assumption because a whole rule is written to combat it.

Reply:
@ShadowBB Agreed. I'll update accordingly.